### PR TITLE
Adding picplace example app

### PR DESCRIPTION
**Pull request by ryanmurakami**
Closes # 

## 📜 Details
[Design Doc]()

[Jira]()

## ✅ Checks

- [ ] CI tests (if present) are passing
- [ ] Adheres to code style for repo
- [ ] Contributor License Agreement (CLA) completed if not a Reddit employee

### CODE REVIEW

File: picplace/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "private": true,
+  "name": "picplace",
+  "version": "0.0.0",
+  "license": "BSD-3-Clause",
+  "type": "module",
+  "dependencies": {
+    "@devvit/protos": "0.8.12-next-2023-04-03-d4cc700c.0",
```
Comment (sp3nx0r) [inline picplace/package.json:8]: can you update to public devvit deps

File: picplace/yarn.lock [hunk 2]
```diff
@@ -0,0 +1,333 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@devvit/plugins@0.8.12-next-2023-04-03-d4cc700c.0":
```
Comment (sp3nx0r) [inline picplace/yarn.lock:5]: also you'll likely want to relock without our internal registry in the mix

**Review by sp3nx0r — CHANGES_REQUESTED** [review]
two asks for ya

**Comment by ryanmurakami** [comment]
Updated deps & registry

**Review by sp3nx0r — APPROVED** [review]
:fish:

**Pull request merged**